services/web/server/src/simcore_service_webserver/projects/_handlers_project_nodes.py

```diff
@@ -471,6 +452,111 @@ async def get_project_services_access_for_gid(request: web.Request) -> web.Respo
         inaccessible_services=project_inaccessible_services,
     )
 
-    return web.json_response(
-        {"data": project_group_access.dict(exclude_none=True)}, dumps=json_dumps
+    return envelope_json_response(project_group_access.dict(exclude_none=True))
+
+
+class _HomePageScreenshot(BaseModel):
+    thumbnail_url: HttpUrl
+    file_url: HttpUrl
+
+
+def _fake_screenshots_factory(
+    request: web.Request, node_id: NodeID
+) -> list[_HomePageScreenshot]:
+    assert request.app[APP_SETTINGS_KEY].WEBSERVER_DEV_FEATURES_ENABLED  # nosec
+    # https://placehold.co/
+    # https://picsum.photos/
+    short_nodeid = str(node_id)[4:]
+    count = int(str(node_id.int)[-1])
+    seed = short_nodeid
+    return [
+        _HomePageScreenshot(
+            thumbnail_url=f"https://placehold.co/170x120?text={short_nodeid}",
+            file_url=f"https://picsum.photos/seed/{seed}/500",
+        )
+        for _ in range(count)
+    ]
+
+
+class _ProjectNodeHomePage(BaseModel):
+    project_id: ProjectID
+    node_id: NodeID
+    screenshots: list[_HomePageScreenshot] = Field(default_factory=list)
+
+
+@routes.get(
+    f"/{VTAG}/projects/{{project_id}}/nodes/-/homepage",
```

Comment:
I am not sure I understand that one. Are you here returning the "homepage" of the project? why not /projects/{project_id}/homepage ? is that not simpler?

Reply:
Using `GET projects/{project_id}/nodes/-/preview` denotes that it returns a list with all `GET projects/{project_id}/nodes/{node_id}/preview` 
I would like to reserve `GET projects/{project_id}/preview` for the moment we need to return